IdentityCore/src/controllers/MSIDRequestControllerFactory.m

```diff
@@ -101,6 +130,86 @@ @implementation MSIDRequestControllerFactory
     return localController;
 }
 
++ (nullable id<MSIDRequestControlling>)v2SilentControllerForParameters:(MSIDRequestParameters *)parameters
+                                                        forceRefresh:(BOOL)forceRefresh
+                                                         skipLocalRt:(MSIDSilentControllerLocalRtUsageType)skipLocalRt
+                                                tokenRequestProvider:(id<MSIDTokenRequestProviding>)tokenRequestProvider
+                                                               error:(NSError *__autoreleasing*)error
+{
+    // Nested auth protocol - Reverse client id & redirect uri
+    if ([parameters isNestedAuthProtocol])
```

Comment:
I think duplicate code from 'v1' method here can be refactored

Reply:
I did not do the partial refactoring because I am planning to move to new `v2` completely. I will do refactoring for duplicate code if the methods needs to be co-exist all the time.